### PR TITLE
Update listener page

### PR DESCRIPTION
**Pull request by SimonScharf**

### CODE REVIEW

**Comment by majacannavo** [comment]
Looks awesome, thanks so much Simon!

**Comment by mylesstokowski** [comment]
Yep looks great, thanks!

File: src/pages/dj.js [hunk 1]
```diff
@@ -10,8 +10,21 @@ import Grid from "@material-ui/core/Grid";
 import Button from "@material-ui/core/Button";
 import AddIcon from "@material-ui/icons/Add";
 import CancelIcon from "@material-ui/icons/Cancel";
+import LoginWidget from "../components/LoginWidget";
```
Comment (Ntare62) [inline src/pages/dj.js:13]: I like that you added loginwidget to the dj page as well

**Review by Ntare62 — COMMENTED** [review]
LGTM. Nice work

**Pull request merged**